src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AppointmentService.kt

```diff
@@ -109,8 +110,14 @@ class AppointmentService(
       prisonApiService.createAppointments(makePrisonAppointment(createAppointmentSpecification)).first()
 
     createAppointmentSpecification.repeat?.let {
+
+      val recurringAppointmentIds: Set<Long> = appointmentCreated.recurringAppointmentEventIds?.toSet() ?: emptySet()
+
+      val appointmentIds =
+        setOf(appointmentCreated.appointmentEventId).union(recurringAppointmentIds)
+
       val recurringAppointment =
-        makeMainRecurringAppointment(appointmentCreated, createAppointmentSpecification.repeat)
+        makeRecurringAppointment(appointmentIds = appointmentIds, repeat = createAppointmentSpecification.repeat)
```

Comment:
That first "=" seems superfluous - does Kotlin need that?

Reply:
Good point will update in the next PR